Module/Rule.Permission/Convert/Methods.ps1

```diff
@@ -172,7 +172,7 @@ function Get-PermissionTargetPath
             $stigString -notmatch $regularExpression.programFileFolder
         }
         {
-            $permissionTargetPath = '%SystemDrive%'
+            $permissionTargetPath = '%SystemDrive%\'
```

Comment:
Instead of adding a back slash here is it possible to use Join-Path where $PermissionTargetPath gets consumed?